synapse/api/room_versions.py

```diff
@@ -293,7 +324,7 @@ class RoomVersionCapability:
         ),
         RoomVersionCapability(
             "restricted",
-            RoomVersions.V8,
+            RoomVersions.V9,
```

Comment:
We probably don't want to do this at the same time, we might want to ensure we get some people upgraded first.

Reply:
My initial feeling is we should probably wait 1 version before doing this, but would be curious of other's thoughts.

Reply:
I think it depends on how the clients are representing this flag to the user. If we enable it then obviously at first very few other servers will be able to join the newly created rooms, which might be fine if the UI makes it clear its experimental?

Reply:
It uses this to determine what room version to upgrade to if attempting to change a room to use restricted join rules for an old room version. This leads me to think we should wait until there's at least some uptake before doing this? 🤷 

![image](https://user-images.githubusercontent.com/517124/132006054-ae01848e-e5d9-4104-a5ae-7ee001e39728.png)

Reply:
After some discussion the decision was to change the preferred version 1 release after v9 is available.

Reply:
```suggestion
            RoomVersions.V8,
```